automation/label-approved-veps.py

```diff
@@ -0,0 +1,110 @@
+import os
```

Comment:
would you use e.g. pep8 to format this file?

Reply:
Thanks, I'll follow up.